components/manager/scripts/validate.py

```diff
@@ -0,0 +1,20 @@
+from cloudify import ctx
+from cloudify_cli import constants
+
+from cloudify.exceptions import NonRecoverableError
+import os
+
+try:
+    security_enabled = ctx.node.properties['security']['enabled']
+except KeyError:
+    security_enabled = False
```

Comment:
For achieving the same intended result,
security_enabled = ctx.node.properties.get('security', {}).get('enabled', False)